app/scripts/controllers/ticketFinder.ctrl.js

```diff
@@ -42,33 +43,22 @@ angular.module('yourStlCourts').controller('TicketFinderCtrl', function (TicketF
   };
 
   ctrl.getDOB = function(){
-    if (!numberOfMunicipalitiesSelectedIsValid()){
-      toaster.pop('error', 'Only 5 municipalities can be selected at any one time.  Please remove some municipalities.');
-    }else {
-      if (ctrl.ticketForm.$valid) {
-        var modalInstance = $uibModal.open({
-          animation: false, //allows focus cursor to stay in input box on edge & IE browsers
-          templateUrl: 'views/lookupSecurity.html',
-          controller: 'lookupSecurityCtrl as ctrl',
-          size: 'sm'
-        });
+    if (ctrl.ticketForm.$valid) {
```

Comment:
Please don't put "ticketForm" on "ctrl". "ticketForm" is literally a DOM form element. Ctrl shouldn't be able to "own" it. Instead, pass it in as a parameter to getDOB()

Reply:
I realize this was there before your current PR, but it's a simple change to be more appropriate and in line with your other new code checking for form validity.